client/src/components/CHeader.style.js

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
```

Comment:
what does css do? i've never used it before!

Reply:
used it in props in the same file.